_data/navigation.yml

```diff
@@ -39,7 +39,28 @@ setup:
       - title: Storage
         url: /setup/install/storage/
       - title: Cloud Providers
-        url: /setup/install/providers/
```

Comment:
So the Cloud Providers node is no longer hyperlinked to setup/install/providers/index.html right? Is that necessary in order to make it collapse/expandable, or can the expand/collapse be done using an arrow icon only, such that I could click into the Overview from this heading __or__ expand it to see the sub-headings?

Reply:
This is technically feasible I think but may prove a bit confusing and fiddly.  The click target for expand / collapse would be small, particularly for tablet users, and the behaviour of a chevron working differently to its label deviates from other examples I can find (e.g. side nav on https://kubernetes.io/docs/home/).  Open to discussion here, let me know if you have more thoughts on this.